angular-hometask/src/app/core/breadcrumbs/breadcrumbs.component.html

```diff
@@ -1 +1,6 @@
-<p>Courses</p>
```

Comment:
Pls make root div or smth like this

Reply:
Done 093fb6e5337b9071065cad01394a1dd84974eabe